model/deployment.go

```diff
@@ -17,39 +17,33 @@ type deployment struct {
 	Args      []string `yaml:"args"`
 }
 
-//Deployment returns a k8 deployment for a cloud native environment
+//Deployment returns a k8 deployment
 func (dev *Dev) Deployment() (*appsv1.Deployment, error) {
-	file, err := os.Open(dev.Swap.Deployment.File)
+	return dev.loadDeployment()
+}
+
+//DevDeployment returns a k8 deployment modified with  a cloud native environment
+func (dev *Dev) DevDeployment() (*appsv1.Deployment, error) {
+	d, err := dev.loadDeployment()
 	if err != nil {
 		return nil, err
 	}
 
-	dec := k8Yaml.NewYAMLOrJSONDecoder(file, 1000)
-	var d appsv1.Deployment
-	dec.Decode(&d)
-
-	d.GetObjectMeta().SetName(dev.Name)
 	labels := d.GetObjectMeta().GetLabels()
 	if labels == nil {
 		labels = map[string]string{"cnd": dev.Name}
 	} else {
 		labels["cnd"] = dev.Name
 	}
 	d.GetObjectMeta().SetLabels(labels)
-	if d.Spec.Selector == nil {
-		d.Spec.Selector = &metav1.LabelSelector{
-			MatchLabels: map[string]string{"cnd": dev.Name},
-		}
-	} else {
-		d.Spec.Selector.MatchLabels["cnd"] = dev.Name
-	}
-	d.Spec.Template.GetObjectMeta().SetName(dev.Name)
+
```

Comment:
maybe in another PR, but could we define a flag to deactivate (or not) every kind on healthcheck?